.travis.yml

```diff
@@ -31,6 +31,11 @@ language: python
 python:
   - 2.7
 
+env:
+  - PYTEST="tests/test_views.py"
+  - PYTEST="tests/test_workflows.py"
+  - PYTEST="--ignore tests/test_views.py --ignore tests/test_workflows.py"
```

Comment:
No worries! I will fix this eventually, as long as `Bibtex` doesn't kill me first...